tests/robot-tests/tests/admin/bau/create-amendment-check-permalink.robot

```diff
@@ -0,0 +1,198 @@
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+Go to Table Tool page
+    [Tags]  HappyPath
+    environment variable should be set  PUBLIC_URL
+    user goes to url  %{PUBLIC_URL}/data-tables
+    user waits until h1 is visible  Create your own tables online
+
+Select "Pupil Exclusions" publication
+    [Tags]  HappyPath
+    user opens details dropdown    Test theme
+    user opens details dropdown    ${TOPIC_NAME}
+    user clicks radio      ${PUBLICATION_NAME}
+    user clicks element    id:publicationForm-submit
+    user waits until h2 is visible  Choose a subject
+    user checks previous table tool step contains  1   Publication   ${PUBLICATION_NAME}
+
+Select subject "Duration of fixed exclusions"
+    [Tags]  HappyPath
+    user clicks radio   ${SUBJECT_NAME}
+    user clicks element   id:publicationSubjectForm-submit
+    user waits until h2 is visible  Choose locations
+    user checks previous table tool step contains  2    Subject     ${SUBJECT_NAME}
+
+Select Location Country, England
+    [Tags]  HappyPath
+    user opens details dropdown     National
+    user clicks checkbox    England
+    user clicks element     id:locationFiltersForm-submit
+    user waits until h2 is visible  Choose time period
+    user checks previous table tool step contains  3   National     England
+
+Select Start date and End date
+    [Tags]  HappyPath
+    user selects from list by label  id:timePeriodForm-start   2020 Week 14
+    user selects from list by label  id:timePeriodForm-end     2020 Week 14
+    user clicks element     css:#timePeriodForm-submit
+    user waits until h2 is visible  Choose your filters
+    user waits until page contains element   id:filtersForm-indicators
+    user checks previous table tool step contains  4    Start date    2020 Week 14
+    user checks previous table tool step contains  4    End date      2020 Week 14
+
+Select Indicators
+    [Tags]  HappyPath
+    user clicks indicator checkbox   Number of open settings
+
+Select Filters
+    [Tags]   HappyPath
+    user opens details dropdown   Date
+    user clicks category checkbox   Date   30/03/2020
+    user checks category checkbox is checked   Date   30/03/2020
+
+Create table
+    [Tags]  HappyPath
+    user clicks element     css:#filtersForm-submit
+    user waits until results table appears     60
+
+Validate results table column headings
+    [Tags]  HappyPath
+    user checks table column heading contains  1   1   2020 Week 14
+
+Generate the peramlink
+  [Tags]  HappyPath
+  user clicks button  Generate permanent link
+  user waits until page contains element  xpath://*[@data-testid="permalink-generated-url"]
+  ${PERMA_LOCATION_URL}=  Get Text  xpath://*[@data-testid="permalink-generated-url"]
+  Set Suite Variable  ${PERMA_LOCATION_URL}
+
+Go to permalink
+    [Tags]  HappyPath
+    user goes to url  ${PERMA_LOCATION_URL}
+    user waits for page to finish loading
```

Comment:
`user waits for page to finish loading` is just a 0.2 sec sleep - and you're already waiting for the heading to load after this keyword, so I'd remove this.